src/GoodsList.scss

```diff
@@ -0,0 +1,10 @@
+.GoodsList {
+  list-style: none;
+
+  padding: 30px;
+  width: max-content;
+
+  border: 1px solid blueviolet;
+
+  background-color: rgb(170, 255, 227);
+}
```

Comment:
End of file is mising))

Reply:
Well, thanks, but it is an end of file in the VSCode. In fact, linter wouldn't let me to commit the code if there wasn't end of file)))

Reply:
Niether do I) But still , it's missing)